lib/index.js

```diff
@@ -113,4 +158,66 @@ Drip.prototype.normalize = function(obj){
   }
 
   return ret;
-}
+};
+
+/**
+ * Set the partner request limit.
+ *
+ * @param {Function} headers
+ * @param {Function} completion
+ * @api private
+ */
+
+Drip.prototype.setLimit = function(headers, completion) {
+  var appId = this.settings.account;
+  var key = ['drip', appId].join(':');
+  var redis = this.redis();
+
+  var limit = {
+    remaining: headers['x-ratelimit-remaining'],
+    reset: headers['x-ratelimit-reset'] || Date.now() + (1000 * 60 * 5)
+  };
+
+  redis.set(key, JSON.stringify(limit), function(err, res) {
```

Comment:
We should probably set an expiration on these keys so they don't live forever. An hour or 2 should be good I think?

https://github.com/segment-integrations/integration-intercom/blob/master/lib/index.js#L280